warehouse/query-core/src/main/java/datawave/query/jexl/visitors/ExecutableDeterminationVisitor.java

```diff
@@ -714,51 +711,7 @@ public Object visit(ASTNENode node, Object data) {
     
     @Override
     public Object visit(ASTReference node, Object data) {
-        STATE state;
-        QueryPropertyMarker.Instance instance = QueryPropertyMarker.findInstance(node);
-        // until we implement an ivarator that can handle an ExceededTermThreshold node, and ensure that the JexlContext gets
-        // _ANYFIELD_ values, then we cannot execute these nodes
-        if (instance.isType(ExceededTermThresholdMarkerJexlNode.class)) {
-            state = STATE.NON_EXECUTABLE;
-            if (output != null) {
-                output.writeLine(data, node, "( Exceeded Term Threshold )", state, true);
-            }
-        }
-        // if an ivarator then return true, else check out children
-        else if (instance.isAnyTypeOf(ExceededValueThresholdMarkerJexlNode.class, ExceededOrThresholdMarkerJexlNode.class)) {
-            state = STATE.EXECUTABLE;
-            if (output != null) {
-                output.writeLine(data, node, "( Exceeded Or / Value Threshold )", state, true);
-            }
-        }
-        // if a delayed predicate, then this is not-executable against the index by choice
-        else if (instance.isAnyTypeOf(ASTDelayedPredicate.class, ASTEvaluationOnly.class)) {
-            if (isNoFieldOnly(node)) {
-                state = STATE.IGNORABLE;
-            } else if (instance.isType(ASTEvaluationOnly.class) && isNonEvent(node)) {
-                state = STATE.ERROR;
-            } else {
-                state = STATE.NON_EXECUTABLE;
-            }
-            if (output != null) {
-                output.writeLine(data, node, "( delayed/eval only predicate )", state, true);
-            }
-        }
-        // if we got to a bounded range, then this was expanded and is not executable against the index
-        else if (instance.isType(BoundedRange.class)) {
-            state = STATE.NON_EXECUTABLE;
-            if (output != null) {
-                output.writeLine(data, node, "( bounded range )", state, true);
-            }
-        } else if (instance.isType(IndexHoleMarkerJexlNode.class)) {
-            state = STATE.NON_EXECUTABLE;
-            if (output != null) {
-                output.writeLine(data, node, "( index hole )", state, true);
-            }
-        } else {
-            state = allOrNone(node, data);
-        }
-        return state;
+        return allOrNone(node, data);
```

Comment:
I thought ASTReference went away around expressions and AND statement?  Should this visit function simply be dropped?